src/bootstrap/src/core/build_steps/test.rs

```diff
@@ -1397,6 +1398,52 @@ impl Step for RunMakeSupport {
     }
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+pub struct RunMakeSupportTest {
+    host: TargetSelection,
+}
+
+impl Step for RunMakeSupportTest {
+    type Output = ();
```

Comment:
Looking at the `RunMakeSupportTest` type, we only want this to run on the host target. If that's the case, let's add this:

```suggestion
    type Output = ();
    const ONLY_HOSTS: bool = true;
```

Reply:
Thanks.